CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 2.8...3.5 FATAL_ERROR)
 
-project(isle CXX)
+project(isle CXX RC)
```

Comment:
I can't find any evidence that `RC` is a valid (or necessary) language to declare:

![image](https://github.com/isledecomp/isle/assets/34096995/48295dc0-7d09-4064-ba5d-7e877176929a)
(https://cmake.org/cmake/help/latest/command/project.html)

Reply:
I added it because no matter how many times I execute `cmake --build .`, `ISLE.EXE` keeps getting rebuilt. But it didn't help and I forgot to remove it.

[Looks like it is an "internal" language](https://github.com/Kitware/CMake/blob/20f07467a740e5eb407689b8c2072dc63763f806/Modules/Platform/Windows-MSVC.cmake#L510)